src/lib/registration/sw_mgmt.rb

```diff
@@ -481,14 +481,28 @@ def self.copy_old_credentials(source_dir)
       # SMT uses extra ACL permissions, make sure they are kept in the copied file,
       # (use "cp -a ", ::FileUtils.cp(..., preserve: true) cannot be used as it preserves only
       # the traditional Unix file permissions, the extended ACLs are NOT copied!)
-      `cp -a #{Shellwords.escape(file)} #{Shellwords.escape(new_file)}`
+      Yast::Execute.locally!("cp", "-a", file, new_file)
```

Comment:
Good that I wrote the comment above, I was just about to suggest using `FileUtils.cp` here. :smiley: